ui/Screen/ProjectCreation.svelte

```diff
@@ -226,7 +226,6 @@
   .wrapper {
     display: flex;
     justify-content: center;
-    margin: 92px 0 72px 0;
```

Comment:
That margin might have been useful:

![margin](https://user-images.githubusercontent.com/158411/81515154-ed790280-9332-11ea-8507-51c1a706056c.gif)